src/graph/ApolloGraphClient.ts

```diff
@@ -92,26 +113,41 @@ export class ApolloGraphClient implements GraphClient {
         return this.client.query<T>({
                 query,
                 variables,
-                ...options,
+                ...queryOptions,
             })
             .then(result => callback(result));
     }
 
+    public mutate<T, Q>(options: MutationOptions<Q> | string): Promise<T> {
+        if (typeof options === "string") {
+            options = {
+                name: options,
+            };
+        }
+        const m = graphql.mutate({
+            mutation: options.mutation,
+            path: options.path,
+            name: options.name,
+            moduleDir: trace.get()[1].getFileName(),
```

Comment:
oh that is _sneaky_.

I like it, although we have to be careful to never wrap these functions internally -- I'd rather make that clear. let me put in a commit